README.adoc

```diff
@@ -258,8 +258,8 @@ Refresh `http://localhost:9080/config` to see the dynamic changes.
 
 Play with this application by changing configuration values for each property in the `CustomConfigSource.json` file.
 
-For example, change `io_openliberty_guides_inventory_inMaintenance` from `false` to `true`, then try to access `http://localhost:9080/inventory/hosts` again,
-you can see a message that says "Service is temporarily down for maintenance".
+For example, change `io_openliberty_guides_inventory_inMaintenance` from `false` to `true`, then try to access `http://localhost:9080/inventory/hosts` again.
+The following message displays: `Service is temporarily down for maintenance`.
 
```

Comment:
Throughout, I made changes to introduce the message. For instance, I added "following message" because this will help vision-impaired users know what to expect next.